i18n/en/docusaurus-plugin-content-docs/current/devices/tariffs.mdx

```diff
@@ -0,0 +1,338 @@
+---
+sidebar_position: 4
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Tariffs
+
+You can configure electricity tariffs for grid consumption, feed-in and CO₂ intensity. This allows evcc to calculate your savings and automatically shift charging to cheaper times.
+
+## Fixed Electricity Price
+
+The simplest case is fixed values for grid consumption (`grid`) and feed-in (`feedin`).
+
+
+```yaml
+tariffs:
+  currency: EUR # (default EUR)
+  grid:
+    type: fixed
+    price: 0.294 # 29,4 ct/kWh
```

Comment:
Können wir auf die Erläuterung verzichten? Die verwirrt mehr als das sie hilft. Alternativ in kurz: `# €`. Siehe https://github.com/evcc-io/evcc/pull/13801#issuecomment-2100218163

Reply:
Ist angepasst.